examples/common/mod.rs

```diff
@@ -36,34 +43,53 @@ pub(crate) fn qos_from_str(s: &str) -> Result<mqtt3::proto::QoS, String> {
 
 pub(crate) const PACKET_STATS_WINDOW_SIZE: usize = 60;
 
+type History = heapless::HistoryBuffer<u128, PACKET_STATS_WINDOW_SIZE>;
+
 #[allow(dead_code)]
 pub(crate) struct PacketStats {
     start_time: std::time::Instant,
     current: u128,
-    history: heapless::HistoryBuffer<u128, PACKET_STATS_WINDOW_SIZE>,
+    bytes: u128,
+    message_history: History,
+    byte_history: History,
 }
 
 impl PacketStats {
     #[allow(dead_code)]
-    pub(crate) fn count(&mut self, num: usize) {
+    pub(crate) fn log_metrics(&mut self, num: usize, bytes: usize) {
```

Comment:
`count` is correct. It doesn't always log.

Reply:
Will rename back :)